RELEASE.md

```diff
@@ -6,4 +6,39 @@ No manual creation of a midstream `release-v1.x` branch is needed. The nightly J
 
 ## Enable CI for the release branch
 
-TODO
+* Create a fork and clone of https://github.com/openshift/release into your `$GOPATH`
+* On your `openshift/knative-serving` root folder checkout the new `release-vX.Y` branch and run:
+
+```bash
+# Invoke CI config generation, and mirroring images
+
+make update-ci
+```
+
+The above `make update-ci` adds new CI configuration to the `openshift/release` repository and afterwards shows which new files were added, like below:
+
+```bash
+┌────────────────────────────────────────────────────────────┐
+│ Summary...                                                 │
+└────────────────────────────────────────────────────────────┘
+│─── Modified files in /home/knakayam/.go/src/github.com/openshift/release
+core-services/image-mirroring/knative/mapping_knative_v1_3_quay
+│─── New files in /home/knakayam/.go/src/github.com/openshift/release
+ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v1.3__410.yaml
```

Comment:
Pls add a note that image promotion is done in this jobs (meant for someone who is new to this).

Reply:
Could you please bring the topic into release sig? 
I actually don't have much knowledge and this RELEASE.md is just copied from eventing's https://github.com/openshift/knative-eventing/blob/main/RELEASE.md

Reply:
Sure will do.